test/page-detect.js

```diff
@@ -370,3 +370,11 @@ test('isTrending', urlMatcherMacro, pageDetect.isTrending, [
 	'https://github.com/watching',
 	'https://github.com/jaredhanson/node-trending/tree/master/lib/trending'
 ]);
+
+test('isUserProfile', urlMatcherMacro, pageDetect.isUserProfile, [
+	'https://github.com/sindresorhus'
+], [
+	'https://github.com/settings',
```

Comment:
Can you also check for `http://github.com/`? I think my ‘includes()` will think the homepage is a profile too.

Reply:
Aha, you're right, great spot! I'm going to add a check for a path existing, I was going to add one of these, have you got any preference? (Personally I prefer `Boolean()`)
```js
Boolean(getCleanPathname())
// or
!!getCleanPathname()
```

Reply:
!! is not allowed by xo. You can also check its length or use a single regex like `/[^/]{1,}/` to test for both length and single-level.